js/packages/stream/src/textSchema.ts

```diff
@@ -0,0 +1,15 @@
+// OpenAPI object definition for a simple text based reponse.
```

Comment:
is this intentional?

```suggestion
// JSON schema for a simple text based reponse.
```